aospy/data_loader.py

```diff
@@ -89,11 +88,7 @@ def set_grid_attrs_as_coords(ds, set_time_vars=True):
     Dataset
         Dataset with grid attributes set as coordinates
     """
-    if set_time_vars:
-        int_names = GRID_ATTRS.keys()
-    else:
-        int_names = (set(GRID_ATTRS.keys()) -
-                     set(TIME_VAR_STRS))
+    int_names = GRID_ATTRS.keys()
```

Comment:
Could just use `GRID_ATTRS.keys()` directly rather than defining `int_names`

Reply:
I think this was the way it was prior to #199 (I think because of line length in the following line).  Happy to change it though.